cytotable/convert.py

```diff
@@ -1440,6 +1408,21 @@ def convert(  # pylint: disable=too-many-arguments,too-many-locals
             if chunk_size is None
             else chunk_size
         )
+        page_keys = (
+            cast(dict, config[preset]["CONFIG_PAGE_KEYS"])
+            if page_keys is None
+            else page_keys
+        )
+
+    # Raise an exception for scenarios where one configures CytoTable to join
+    # but does not provide a pagination key for the joins.
+    if join and (page_keys is None or "join" not in page_keys.keys()):
+        raise CytoTableException(
+            (
+                "When using join=True one must pass a 'join' pagination key "
+                "in the page_keys parameter."
```

Comment:
Can you answer the question "why?" in this error message? Or perhaps provide an example?

Reply:
Thanks - I'll add additional context as you suggested. Because the join composite table may include variable columns contingent on the join SQL there is a special key which may be specified in CytoTable to help perform keyset pagination on the join table. If there is no 'join' key specified when attempting to join we except here to help prevent further operations.